File: C++/Euler1/euler1.cpp
#include <iostream>
using namespace std;

// Function Name
int numbers();

int main()
{
    //Clock stuff
    std::clock_t start;
    double duration;
    
    start = std::clock();
    
    //Actual Algorithm Here
    int sum = 0;
    for (int i =1; i<1000; i++) {
        if (i%3 == 0 || i%5 == 0) {
            sum +=i;
        }
    }
    
    //Finish Clock Stuff
    duration = ( std::clock() - start ) / (double) CLOCKS_PER_SEC;
    
    cout<< "Elapsed: "<< duration << " seconds"<< endl;
    
    //Print Answer to Algorithm
    cout << "The Sum of Multiples of 3 and 5 below 1000 is: " << sum << endl;
    
    return 0;
}

File: Fortran/euler1/euler1.f90
program EULER1
implicit none

	real :: start,finish
	call cpu_time(start)
	print *, "Sum of multiples of 3 and 5 below 1,000: ", sum()
	call cpu_time(finish)

	print'("Elapsed time: ", F9.6, " seconds")', finish - start

contains


	!!Finds Sum of multiples of 3's and 5's below 1,000
	integer function sum()
		implicit none
		integer :: i, total
		total = 0
		do i = 1, 999
			if ((mod(i,3) == 0) .or. (mod(i,5) == 0)) then
				total = total + i
			endif
		enddo

		sum = total
	end function sum

end program EULER1
